src/page-wrappers/build/me-model/emodel.selection.tsx

```diff
@@ -104,6 +133,45 @@ export default function EmodelSelection({ params, searchParams }: Props) {
             Select e-model
           </Btn>
         )}
+        onRow={(row) => {
+          if (checkSelectedEmodelBlackList(row))
+            // this new line in the attribute is required to be displayed in two lines
+            return {
+              'black-listed': `This e-model cannot be combined 
+              with any morphology for now.
+              `,
+            } as HTMLAttributes<any> & TdHTMLAttributes<any>;
+          return {};
+        }}
+        rowClassName={(row: IEModel) => {
+          return checkSelectedEmodelBlackList(row)
+            ? classNames(
```

Comment:
That the reason why I don't like TailWind so much  ;-)

But, you can keep this code as is. It's not a problem for me.